src/backend/distributed/utils/citus_depended_object.c

```diff
@@ -314,13 +313,13 @@ GetCitusDependedObjectArgs(int pgMetaTableVarno, int pgMetaTableOid)
 
 
 /*
- * DistOpsHasInvalidObject returns true if any address in the given node
- * is invalid; otherwise, returns false. If ops is null or it has no
+ * DistOpsHasAllInValidObject returns true if all addresses in the given node
+ * are invalid; otherwise, returns false. If ops is null or it has no
  * implemented address method, we return false. We also have some dist ops
  * for which we should not validate and return false.
  */
 bool
-DistOpsHasInvalidObject(Node *node, const DistributeObjectOps *ops)
+DistOpsHasAllInValidObject(Node *node, const DistributeObjectOps *ops)
```

Comment:
shouldn't this be called `DistOpsHasAnyValidObject` ? All variables names as well should be adjusted